CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## Release Highlights
 
 ## Important Notes
-
+- [#799](https://github.com/oauth2-proxy/oauth2-proxy/pull/799) Use comma separated multiple values for header. Now multi-values headers like 
```

Comment:
I think we can just move this under `## Changes since v6.1.1` since we didn't have multiple value headers before v6.1.1, 
Please just add the following in the list for below, around line 70 at the moment
```
- [#799](https://github.com/oauth2-proxy/oauth2-proxy/pull/799) Use comma separated multiple values for header (@lilida)
```